spec/factories.rb

```diff
@@ -20,19 +21,18 @@
     end
   end
 
+  factory :step_by_step_page_with_steps_but_no_draft, parent: :step_by_step_page_with_steps do
+    draft_updated_at { nil }
+  end
+
   factory :published_step_by_step_page, parent: :step_by_step_page_with_steps do
-    draft_updated_at { 3.hours.ago }
     published_at { 3.hours.ago }
     status { "published" }
   end
 
-  factory :draft_step_by_step_page, parent: :step_by_step_page_with_steps do
-    draft_updated_at { 1.day.ago }
-    status { "draft" }
-  end
+  factory :draft_step_by_step_page, parent: :step_by_step_page_with_steps
```

Comment:
Is there a need for this factory?  It seems to be identical to `step_by_step_page_with_steps`.

Reply:
It is identical - think of one as an alias for the other. Each are referenced in several test files, in whichever way reads most naturally, with the emphasis either on the fact it's a draft or the fact it has steps.

It feels like it would be nice to go in at some point and reduce to just one (`draft_step_by_step_page_with_steps`?), but I'd have to make a few changes in around 7 files and some of the original intent might be lost. I'd be tempted to leave as-is, giving the two implementations freedom to grow apart if needs arise.

Reply:
FactoryBot allows you to [alias factories](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#aliases).  This may be a more optimal way of having both names for the same factory.

Reply:
Nice!